### PR TITLE
Release notes for RC3

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Release notes for RC3

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: endgame

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by artursouza** [comment]
Please, do your changes on HackMD. Link is in the maintainer's Discord channel.

File: docs/release_notes/v1.0.0-rc.3.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+
+* Pub/Sub [per message time-to-live](https://v1-rc3.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-message-ttl/)
+Dapr enables per-message time-to-live (TTL) for all pub/sub components. Applications can set time-to-live per message, and subscribers will not receive those messages after expiration.
+* Sidecar to app authentication
+* [MySQL output binding](https://v1-rc3.docs.dapr.io/operations/components/setup-bindings/supported-bindings/mysql/)
+* [MySQL state store](https://v1-rc3.docs.dapr.io/operations/components/setup-state-store/supported-state-stores/setup-mysql/)
+* Component metadata's random value via `{uuid}` marker
+* [Upgrade CLI command to upgrade the Dapr control plane](https://github.com/dapr/cli#upgrade-dapr-on-kubernetes)
+* Numerous improvements to the [Dapr Dashboard](https://github.com/dapr/dashboard) 
+* Added [dapr-distributed-calendar sample]9https://github.com/dapr/samples/tree/master/dapr-distributed-calendar)
```
Comment (eNeRGy164) [inline docs/release_notes/v1.0.0-rc.3.md:22]: Typo

```suggestion
* Added [dapr-distributed-calendar sample](https://github.com/dapr/samples/tree/master/dapr-distributed-calendar)
```

File: docs/release_notes/v1.0.0-rc.3.md [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+
+The .NET SDK has several major feature additions and impactful changes in this release. We recommend every user read this [migration guide](https://github.com/dapr/dotnet-sdk/issues/569) as a supplement to the release notes.
+
+- **RESOLVED** Remove Query all statenames from IActorStateManager because it does not behave as expected [53](https://github.com/dapr/dotnet-sdk/issues/53)
+
+- **FIXED** Changed signatures to use IReadOnlyDictionary<TKey, TValue> over Dictionary<TKey, TValue> in public APIs [395](https://github.com/dapr/dotnet-sdk/issues/395)
+
+- **FIXED** Actor Runtime should not report Exception text in HTTP responses [410](https://github.com/dapr/dotnet-sdk/issues/410)
+
+- **Add** support for metada in pub/sbu [425](https://github.com/dapr/dotnet-sdk/issues/425)
```
Comment (eNeRGy164) [inline docs/release_notes/v1.0.0-rc.3.md:182]:
```suggestion
- **Add** support for metadata in pub/sub [425](https://github.com/dapr/dotnet-sdk/issues/425)
```

File: docs/release_notes/v1.0.0-rc.3.md [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+
+- **RESOLVED** Simplifying Service Invocation API surface [549](https://github.com/dapr/dotnet-sdk/issues/549)
+
+- **RESOLVED** Expose Dapr API token to users as a settable value [551](https://github.com/dapr/dotnet-sdk/issues/551)
+
+- **FIX** DaprClient now implements IDisposable so that networking resources can be reclaimed deterministically. [559](https://github.com/dapr/dotnet-sdk/issues/559)
+
+### Go SDK
+
+- **RESOLVED** Remove unstable dependencies. [136](https://github.com/dapr/go-sdk/issues/136)
```
Comment (halspang) [inline docs/release_notes/v1.0.0-rc.3.md:496]: Is this change actually in the release? I don't see a corresponding commit/PR and it's still in To Do in Milestone 3.

Comment (yaron2) [reply docs/release_notes/v1.0.0-rc.3.md:496]: It's been closed. I'll remove it from the hack/md.

File: docs/release_notes/v1.0.0-rc.3.md (hunk 2 above)
Comment (yaron2) [reply docs/release_notes/v1.0.0-rc.3.md]: The source of this file is hosted someplace else and synced here. I fixed the source based on your suggestion. Thanks!

File: docs/release_notes/v1.0.0-rc.3.md (hunk 1 above)
Comment (yaron2) [reply docs/release_notes/v1.0.0-rc.3.md]: The source of this file is hosted someplace else and synced here. I fixed the source based on your suggestion. Thanks!

File: docs/release_notes/v1.0.0-rc.3.md [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+
+- **FIXED** error handling for service invocation with HTTP client and gRPC service. [461](https://github.com/dapr/java-sdk/issues/461)
+
+- **ADDED** examples on how to write Unit tests with DaprClient and Actors. [470](https://github.com/dapr/java-sdk/issues/470)
+
+- **FIXED** order of State key-value params. [472](https://github.com/dapr/java-sdk/issues/472)
+
+
+### Python SDK
+
```
Comment (wcs1only) [inline docs/release_notes/v1.0.0-rc.3.md:288]:
```suggestion
python-sdk version is 1.0.0rc3 and is available through pip/pypi
```

File: docs/release_notes/v1.0.0-rc.3.md [hunk 5]
```diff
@@ -0,0 +1,552 @@
...
+- **FIXED** error handling for service invocation with HTTP client and gRPC service. [461](https://github.com/dapr/java-sdk/issues/461)
+
+- **ADDED** examples on how to write Unit tests with DaprClient and Actors. [470](https://github.com/dapr/java-sdk/issues/470)
+
+- **FIXED** order of State key-value params. [472](https://github.com/dapr/java-sdk/issues/472)
+
+
+### Python SDK
+python-sdk version is 1.0.0rc3 and is available through pip/pypi
```
Comment (wcs1only) [inline docs/release_notes/v1.0.0-rc.3.md:288]:
```suggestion

Python SDK version is 1.0.0rc3 and is available through pip/pypi


```

**Review by yaron2 — APPROVED** [review]

**Pull request merged**